tests/utils/deployment/deployment.go

```diff
@@ -265,3 +267,20 @@ func RedefineWithPVC(deployment *v1.Deployment, name string, claim string) *v1.D
 
 	return deployment
 }
+
+func RedefineWithResources(deployment *v1.Deployment, limit string, req string) {
+	for i := range deployment.Spec.Template.Spec.Containers {
+		deployment.Spec.Template.Spec.Containers[i].Resources = corev1.ResourceRequirements{
+			Limits: corev1.ResourceList{
+				corev1.ResourceCPU: resource.MustParse(limit),
+			},
+			Requests: corev1.ResourceList{
+				corev1.ResourceCPU: resource.MustParse(req),
+			},
+		}
+	}
+}
+
+func RedefineWithRunTimeClass(deployment *v1.Deployment, rtc *nodev1.RuntimeClass) {
```

Comment:
Why passing a pointer to nodev1.RunTimeClass? You only the name.
```
func RedefineWithRunTimeClass(deployment *v1.Deployment, runtimeClassName string) {
```

Reply:
Fixed, thanks